presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreen.kt

```diff
@@ -111,6 +133,26 @@ fun EventScreen(
             }
         }
     )
+
+    DateTimeDialog(
+        state = calendarStateTimeDate,
+        selection = DateTimeSelection.DateTime(
+            selectedDate = ZonedDateTime.now().toLocalDate(),
+            selectedTime = ZonedDateTime.now().toLocalTime(),
+        ) { localDateTime ->
+            val zoneId = ZoneId.systemDefault()
+            val zonedDateTime = ZonedDateTime.of(localDateTime, zoneId)
+
+            if(isStartDateTime) {
+                eventScreenEvent(EventScreenEvent.OnStartTimeDuration(zonedDateTime))
+                eventScreenEvent(EventScreenEvent.OnStartDateDuration(zonedDateTime))
+            }
+            else {
+                eventScreenEvent(EventScreenEvent.OnEndTimeDuration(zonedDateTime))
+                eventScreenEvent(EventScreenEvent.OnEndDateDuration(zonedDateTime))
+            }
+        }
+    )
```

Comment:
@MKiperszmid @philipplackner In this DateTimeDialog I am setting both the time and the date for the start and end duration. So when the dialog opens the user will set the date and time in one go.

I have added a boolean flag to indicate which should be set the start or the end duration.